.github/workflows/ver3-unit-test.yml

```diff
@@ -0,0 +1,27 @@
+name: 3.0.0 Unit Test
```

Comment:
I would probably just call the file unit-test.yml

Reply:
No problem.